modules/service/ranking-service.js

```diff
@@ -8,13 +8,17 @@ class RankingService {
     initialize() {
         this.implementation = new KadIdentityRanking();
         this.implementation.initialize(this.logger);
+        // Object.keys(implementations).forEach((key, index) => {
```

Comment:
Remove comments

Reply:
Update comments into TODO so we don't forget that we have add some changes here. [Commit with changes](https://github.com/OriginTrail/ot-node/pull/1698/commits/8b29a24c0769af816006e7a59684e5485ae0213a)